src/i18n.ts

```diff
@@ -27,7 +27,7 @@ export function findLocale() {
   let gridTheme = enUS;
   let images = { benefits };
 
-  const detectedLocale = detect(fromUrl("lang") /*fromNavigator(), */);
```

Comment:
can we keep fromUrl in there too? so we have a way to manually override and test other languages?